packages/tools/client-debugger/client-debugger-view/src/test/app/App.tsx

```diff
@@ -368,3 +387,22 @@ function CounterView(props: CounterViewProps): React.ReactElement {
 
 	return sharedCounter === undefined ? <Spinner /> : <CounterWidget counter={sharedCounter} />;
 }
+
+interface EmojiButtonViewProps {
+	emojiCellHandle: IFluidHandle<SharedCell<boolean>>;
+}
+
+function EmojiButtonView(props: EmojiButtonViewProps): React.ReactElement {
+	const { emojiCellHandle } = props;
+
+	const [emojiCell, setEmojiCell] = React.useState<SharedCell<boolean> | undefined>();
```

Comment:
nitpick: this type is referred to in props as `emojiCellHandle` but referred to here as `emojiCell`